kustomize/kustomization.yaml

```diff
@@ -23,11 +23,11 @@ components:
 # - components/network-policies
 # - components/non-public-frontend
 # - components/service-accounts
+# - components/alloydb
```

Comment:
FYI: While merging conflicts, I put `components/alloydb` under `components/service-accounts`. This is intentional. `components/alloydb` modifies the `cartservice` ServiceAccount — so it needs to come after (to avoid Kustomize errors). :)